cmd/trickster/main.go

```diff
@@ -84,6 +85,19 @@ func main() {
 	}
 
 	metrics.Init()
+
+	// Register Tracing Configurations
+	tracerFlushers, err := tr.RegisterAll(config.Config)
+	if err != nil {
+		log.Fatal(1, "tracing registration failed", log.Pairs{"detail": err.Error()})
```

Comment:
Do we want this to be fatal? I feel like we probably want to whine about it but fail open to not block the main functionality.

`					return nil, fmt.Errorf("invalid tracing implementation [%s] for tracing config [%s]", tc.Implementation, oc.TracingConfigName)`

like if someone goofs the config value do we want to block trickster as a whole?

Reply:
It's a valid question. I have always been of the mind that if any of our observability tools fail, we fail, but that's more from a SAAS point of view. For a product like this, I think it's viable either way. Need some advice from dad, @jranson ?

Reply:
Any detectable configuration issue is that is due to human error should cause the application to exit fatally during startup, regardless of how insignificant the issue seems. If we start to make exceptions here and there, then we'd have to maintain the list of which features can be fatally configured and which ones cannot. So the all or nothing approach will win here. (resolving)